packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -1520,6 +1521,23 @@ class FlutterView extends ServiceObject {
         });
   }
 
+  Future<Map<String, Uint8List>> getSkSls() async {
```

Comment:
Same here for `SkSLs`.

Reply:
We're limited to a single character here, and `s` is screenshot and `S` is dump semantics tree. We can also explore writing a service extension that editors/IDEs can invoke on behalf of a user, perhaps when they press a button in devtools or intellij.

FYI @jacob314 @devoncarew

Reply:
Oops, this was meant to be on the switch above

Reply:
(unless you were just taking about the member name and not the keypress...)

Reply:
@kenzieschmoll @terrylucas so they're aware of exposing Skia information to tools.

@jonahwilliams, re: the single letter overload in the CLI. Some ideas:

Change to only have single letter abbreviations for very common operations (hot reload, quit, ...). Have an overflow shortcut key which prints a menu of additional available operations - those could be numbered, or types in by short words (`skia`, `dump`, ...)

Or, have all operations have short phrases associated with them (`hot-reload`, `hot-restart`, `dump-render-tree`). You'd have to type in the full text in order to select the operation, but we could offer typing completion in the console so you'd normally only have to type in a few chars manually.

Reply:
@jonahwilliams : ah yes, I was just talking about renaming `getSkSls()` to `getSkSLs()` as that's the name convention from Skia.

Reply:
I played around with this toy program:
```dart
import 'dart:io';

main() {
  stdin.echoMode = false;
  stdin.lineMode = false;
  stdin.listen((List<int> s) {
    print('> $s\n');
  });
}
```
on Linux and Windows. On both it's possible to detect when `ctrl` is pressed while a letter key is pressed. We'd probably have to decode stdin with a custom decoder rather than `AsciiDecoder` though.

(On Linux alt is sometimes part of the same `List<int>` as the character code when it is pressed, and sometimes not. On Windows it's not detectable.)